src/upload.js

```diff
@@ -4,29 +4,42 @@ const axios = require("axios");
 const FormData = require("form-data");
 const fs = require("fs");
 
-const { walkDirectory, trimTrailingSlash } = require("./utils");
+const { walkDirectory, uriSkynetPrefix, trimTrailingSlash } = require("./utils");
 
-function UploadFile(path, opts) {
-  const options = opts.customFilename ? { filename: opts.customFilename } : {};
+const defaultUploadOptions = {
+  portalUrl: "https://siasky.net",
+  portalUploadPath: "/skynet/skyfile",
+  portalFileFieldname: "file",
+  portalDirectoryFileFieldname: "files[]",
+  customFilename: "",
+};
+
+function uploadFile(path, customOptions = {}) {
+  const opts = { ...defaultUploadOptions, ...customOptions };
 
   const formData = new FormData();
+  const options = opts.customFilename ? { filename: opts.customFilename } : {};
   formData.append(opts.portalFileFieldname, fs.createReadStream(path), options);
 
+  // Form the URL.
   const url = `${trimTrailingSlash(opts.portalUrl)}${trimTrailingSlash(opts.portalUploadPath)}`;
 
   return new Promise((resolve, reject) => {
     axios
       .post(url, formData, { headers: formData.getHeaders() })
-      .then((resp) => {
-        resolve(`sia://${resp.data.skylink}`);
+      .then((response) => {
+        resolve(`${uriSkynetPrefix}${response.data.skylink}`);
```

Comment:
Why do we prefix the response with `sia://` ? Is that the common practice between all the SDKs ? What is the purpose of it ?

Reply:
Yeah, we do that in all the SDKs right now. I think it's just to make it more user-friendly. We'll probably change it when we adjust the SDKS to more closely resemble curl calls, which return more information (and the skylink without `sia://`)